jaeger-core/src/main/java/com/uber/jaeger/baggage/DefaultBaggageRestrictionManager.java

```diff
@@ -0,0 +1,42 @@
+/*
+ * Copyright (c) 2017, Uber Technologies, Inc
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal
+ * in the Software without restriction, including without limitation the rights
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in
+ * all copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+ * THE SOFTWARE.
+ */
+
+package com.uber.jaeger.baggage;
+
+import com.uber.jaeger.metrics.Metrics;
+
+public class DefaultBaggageRestrictionManager extends BaggageRestrictionManager {
+  private final BaggageSetter baggageSetter;
+
+  public DefaultBaggageRestrictionManager(Metrics metrics) {
+    this(metrics, DEFAULT_MAX_VALUE_LENGTH);
+  }
+
+  public DefaultBaggageRestrictionManager(Metrics metrics, int maxValueLength) {
+    baggageSetter = BaggageSetter.of(true, maxValueLength, metrics);
+  }
+
+  @Override
+  public BaggageSetter getBaggageSetter(String key) {
```

Comment:
discussed this offline: my suggestion doesn't work because the getter requires a `key`